pages/integrations/sso.md

```diff
@@ -96,6 +96,9 @@ Yes, team maintainers can select whether a user is 'required' to use SSO or whet
 ### Do you support JIT provisioning?
 Yes, we do.
 
+### What **is** JIT provisioning?
```

Comment:
It doesn't look like the bolding shows up in the heading, but I also don't think it's necessary here:

```suggestion
### What is JIT provisioning?
```